### DISCUSSION TITLE
video card for working with Frigate

### DISCUSSION THREAD

**Discussion by customstudioyandexru** [blakeblackshear/frigate#14961] [Ask A Question]
Hello friends. Please prompt what better to use video card to work with Frigate. I5 - 12400f processor without supporting the video core.

**Comment by tearfulnumpty** [comment]
OpenVINO will probably work for you from your CPU. Is there any reason that you're looking for a GPU in addition to that?  

Even so, most NVIDIA GPUs 1000 series or newer will work about equally. The newer ones support later versions of CUDA, but that's about the only real difference.

**Reply by customstudioyandexru** [reply]
I'd like to use CodeProect.ai I think one processor will be hard. Can I help video card Nvidia gtx 1080i? Or do I need other equipment?

**Reply by tearfulnumpty** [reply]
I don't know anything about CodeProject ai. I'm not aware of any internal support that it has from Frigate. 

If you want to use Frigate's object detection, OpenVino on your CPU will probably be fine for ~10 cameras maybe. You can always give it a try before you write it off.

If you insist on getting a GPU when there's a really strong case you don't actually need it... A 1080 will probably be fine. Again, I don't know anything about your CodeProject tool so I can't speak to that.

**Reply by daxliniere** [reply]
Thanks for the info @tearfulnumpty. The non-F variant of the i5-12400 contains the UHD-730 GPU. It's a low-end GPU, so would it be useful to Frigate in any way? I wonder if it's faster to use CPU only.

My particular setup would be 4 or 5 cameras with very little detection other than motion-activated recording.

Thanks in advance!
-Dax.

**Reply by tearfulnumpty** [reply]
Sorry for the late response.  

I wouldn't know with firsthand experience, but I would surely think that would be sufficient for 4 or 5 cameras with proper detection settings.

**Reply by daxliniere** [reply]
All good. I have been running a 4 cam setup for a couple months; motion detection and recording works perfectly (though the setup process for Frigate rather sado/masochistic. Ain't anyone in Linux land heard of a setup wizard?? 🤣)

**Reply by tearfulnumpty** [reply]
Glad to hear it's working!  I can relate to Frigate being unintuitively documented.  The docs are all there, but they only really make sense to someone who already understands the layout of Frigate.  You're definitely not the first to raise a gripe haha!  

For anyone else reading this, the best way to process the docs is simply and truly to read nearly every page of them, even if they aren't really relevant to what step you're trying to do at the moment.  It's the least hassle inducing way to figure things out.  Frigate could really benefit from following the 4 axiom documentation system in my humble opinion: https://docs.divio.com/documentation-system/

**Reply by hawkeye217** [reply]
Users have also found the AI at https://docs.frigate.video very helpful. It's been trained on the documentation and discussions and is much better at Frigate than other major LLMs.

It makes the process of generating a configuration very simple. For example, you can say "I have 5 cameras, I'm using an Intel N100 and want to detect person and car on all of the cameras. Write me a config file that takes advantage of my hardware for video decoding, OpenVINO for object detection, and go2rtc for live view."

**Reply by daxliniere** [reply]
Thanks @tearfulnumpty and @hawkeye217.
By wizard, I was talking more about a fully GUI-driven approach, one that searches the network for applicable devices then allows you to enter credentials and otherwise configure it. Like a hardware NVR box from the likes of Dahua, heck, even the no-name Chinese box I had before that with some low-end embedded Linux OS had a wizard like that.
But I must say, Frigate's approach is no worse than anything else *nix.
haha, I swear you *nix dudes are masochistic and just want to do things the hardest way possible. 😜😊 There's the regular argument "It's so we have more control", but y'all, those parameters could easily go into checkboxes, radio buttons and dropdown menus. 😁

By the way, I've been a software beta tester for the past 20 years and UI/UX is one of my specialties. You can imagine why I say the things I do about Linux/etc being more difficult than it could be. :)

As for the AI configuration tool - I wish I'd known about that when I was doing my setup!

**Reply by NickM-27** [reply]
Making Frigate easy to setup / configure is something we intend to work on in the future, but it is only now starting to become the priority. Each Frigate release has been making large steps towards this being possible in the background.

**Reply by hawkeye217** [reply]
Right, we are planning full UI configuration for an upcoming version. Stay tuned.

**Reply by daxliniere** [reply]
Thanks guys. <3
To be clear, I'm truly grateful you would even make this available for free in any state. 🙏

**Reply by tearfulnumpty** [reply]
> Making Frigate easy to setup / configure is something we intend to work on in the future, but it is only now starting to become the priority. Each Frigate release has been making large steps towards this being possible in the background.

For what it's worth, I *do* think it's easy to configure as it is and I like the config file approach.  A UI config would of course be a welcome addition, but that isn't the issue with the docs for me, and isn't near the most pressing feature that I'd like to see over actual added functionality.  I wouldn't want a UI config to replace the config file as an option either, but I am sure that isn't what you plan on doing.

I just think the docs aren't easy to pick up and understand for first time users given how they're laid out.  The docs definitely have all of the needed information, it just isn't apparent at first usage when compared to other projects in my opinion.  As above, they make sense when someone actually understands the full structure and context of Frigate and I am very appreciative of that!  

Just my hopefully constructive two cents!  Thanks.
